.github/workflows/pkg.pr.new.yml

```
@@ -6,21 +6,30 @@ on:
     types: [submitted]
 
 jobs:
-  deploy-preview:
-    name: Deploy to pkg.pr.new
+  check-changes:
+    name: Check for relevant changes to deploy
     if: github.event.review.state == 'APPROVED' || contains(github.event.pull_request.author_association, 'MEMBER')
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
         with:
           fetch-depth: 2
-      - name: Check for changes to nuqs package
-        id: check-for-changes
+
+      - name: Check for relevant changes
         run: |
           if git diff --quiet HEAD^ HEAD ./packages/nuqs; then
             echo "No changes to nuqs package, skipping preview deployment."
-            echo "skip=true" >> $GITHUB_OUTPUT
+            echo "enable=false" >> $GITHUB_OUTPUT
+          else
+            echo "enable=true" >> $GITHUB_OUTPUT
           fi
+
+  deploy-preview:
+    name: Deploy to pkg.pr.new
+    if: needs.check-changes.outputs.enable == 'true'
```

We need to connect the two jobs again:
```suggestion
    if: needs.check-changes.outputs.enable == 'true'
    needs: check-changes
```